test/README.md

```diff
@@ -92,7 +92,18 @@ make test
     - If you have many new test cases to add, then you may want to
     copy the TestExample project, change the name,replace the test
     examples with your own code, update the build.xml and playlist.xml
-    files to match your new Test class names.
+    files to match your new Test class names. The playlist.xml format 
+    is defined in TestConfig/playlist.xsd.
+
+    - A test can be tagged with following elements:
+      - level:   [sanity|extended] (extended default value)
+      - group:   [functional|openjdk|external|perf|jck|system] (required 
+                 to provide one group per test.)
+      - impl:    [openj9|hotspot] (filter test based on exported JAVA_IMPL 
+                 value; openj9 is the default value)
```

Comment:
I had thought that the lack of a tag for <impl> would mean that the test can be run against both openj9 or hotspot implementations.  Presumably, if openj9 is the default, it means that we have to tag all systemtests and openjdk regression tests at Adopt with both hotspot and openj9?

Reply:
Yes, it's both. I made a mistake, updating the readme now.